Create bytesize_nfcore_proteinfamilies.md (#3118)
nf-core/website: sites/main-site/src/content/events/2025/bytesize_nfcore_proteinfamilies.md

---
title: "Bytesize: nf-core/proteinfamilies"

> subtitle: Evangelos Karatzas, EMBL Cambridge

Reviewer: ```suggestion
subtitle: Evangelos Karatzas, EMBL-EBI, Cambridge
```

type: talk
startDate: "2025-04-08"

> startTime: "13:00+01:00"

Reviewer 2: ```suggestion
startTime: "13:00+02:00"
```

endDate: "2025-04-08"

> endTime: "13:30+01:00"

Reviewer 2: ```suggestion
endTime: "13:30+02:00"
```

locations:
  - name: Online
    links:
      - https://kth-se.zoom.us/j/68390542812
---

> In this weeks bytesize talk, Evanglos ([@vagkaratzas](https://github.com/vagkaratzas)) is going to introduce the nf-core pipeline nf-core/proteinfamilies.

Reviewer: ```suggestion
In this week's bytesize talk, Evangelos ([@vagkaratzas](https://github.com/vagkaratzas)) is going to introduce the nf-core pipeline nf-core/proteinfamilies.
```
